test/inductor/test_compiled_optimizers.py

```diff
@@ -81,7 +164,10 @@ def test_fn(self):
             opt_eager.step()
 
         self.assertEqual(
```

Comment:
Also, could we assert that state is correct? (Or we could land my PR https://github.com/pytorch/pytorch/pull/115680)